os-appium-tests/awsConfiguration.yml

```diff
@@ -0,0 +1,84 @@
+version: 0.1
```

Comment:
even this file is prepared to android , iOS the configurations are different we should make it explicit